sites/website/src/docs/design-systems/fast-frame.md

```diff
@@ -4,228 +4,297 @@ title: The FAST Frame Design System
 
-:::note
-We recommend to add the `<fast-design-system-provider>` at the root of your view with the `use-defaults` attribute. This ensures that all Design System properties are initialized. For more info see the [`use-defaults`](/docs/design-systems/using-the-design-system#use-defaults)
-:::
 
-### Type-ramp
-The FAST type ramp is exposed by the `FASTDesignSystemProvider` as CSS Custom Properties. It organizes the ramp around a _base_ font size and line-height, ascending and descending from the _base_. The CSS Custom Properties that can be used are:
+### FAST Frame Design Tokens
+FAST exposes the following Design Tokens that can be used to configure components stylistically. This section describes the non-color related Design Tokens. For Design Tokens related to color, see the [adaptive color system section](/docs/design-systems/fast-frame#adaptive-color-system)
 
-| Level              | Font Size                       | Line Height                      |
+#### Typography
+| Level              | Font Size Token Name            | Line Height Token Name           |
 |--------------------|---------------------------------|----------------------------------|
-| Minus 2 (smallest) | `--type-ramp-minus-2-font-size` | `--type-ramp-minus-2-line-height`|
-| Minus 1            | `--type-ramp-minus-1-font-size` | `--type-ramp-minus-1-line-height`|
-| Base (body)        | `--type-ramp-base-font-size`    | `--type-ramp-base-line-height`   |
-| Plus 1             | `--type-ramp-plus-1-font-size`  | `--type-ramp-plus-1-line-height` |
-| Plus 2             | `--type-ramp-plus-2-font-size`  | `--type-ramp-plus-2-line-height` |
-| Plus 3             | `--type-ramp-plus-3-font-size`  | `--type-ramp-plus-3-line-height` |
-| Plus 4             | `--type-ramp-plus-4-font-size`  | `--type-ramp-plus-4-line-height` |
-| Plus 5             | `--type-ramp-plus-5-font-size`  | `--type-ramp-plus-5-line-height` |
-| Plus 6 (largest)   | `--type-ramp-plus-6-font-size`  | `--type-ramp-plus-6-line-height` |
-
-### Adaptive Color
+| Minus 2 (smallest) | `typeRampMinus2FontSize`        | `typeRampMinus2LineHeight`       |
+| Minus 1            | `typeRampMinus1FontSize`        | `typeRampMinus1LineHeight`       |
+| Base (body)        | `typeRampBaseFontSize`          | `typeRampBaseLineHeight`         |
+| Plus 1             | `typeRampPlus1FontSize`         | `typeRampPlus1LineHeight`        |
+| Plus 2             | `typeRampPlus2FontSize`         | `typeRampPlus2LineHeight`        |
+| Plus 3             | `typeRampPlus3FontSize`         | `typeRampPlus3LineHeight`        |
+| Plus 4             | `typeRampPlus4FontSize`         | `typeRampPlus4LineHeight`        |
+| Plus 5             | `typeRampPlus5FontSize`         | `typeRampPlus5LineHeight`        |
+| Plus 6 (largest)   | `typeRampPlus6FontSize`         | `typeRampPlus6LineHeight`        |
+-------------------------------------------------
+
+#### Sizing
+- `baseHeightMultiplier`: TODO - Bheston
```

Comment:
@bheston I wasn't sure how to best describe these design tokens - can you take a stab at it?